.github/ISSUE_TEMPLATE/udev_request.yml

```
@@ -0,0 +1,27 @@
+name: Request a udev rule
+description: Request udev rules to help with hardware enablement
+labels: [package-request]
+
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thank you for taking the time to fill out this request!
+  - type: textarea
+    id: describe-bug
+    attributes:
+      label: Describe the package
+      description: Include why you feel this should be built in the ublue-os COPR
+      placeholder: Tell us what you need
+      value: "I'd like to request the package `vim` because ..."
```

```suggestion
      value: "I'd like to request this udev rule because ..."
```
Might want to change this line?